### ISSUE TITLE
Where does ListNodeInfoADM.comments come from?

### ISSUE THREAD

**Issue by benjamingeer** [dasch-swiss/knora-api#1508]
`ListNodeInfoADM` has a member `comments`, which `LIstsResponderADM` populates from `rdfs:comment`. But `knora-base:ListNode` has no cardinality on `rdfs:comment`.

**Comment by subotic** [comment]
Yes, it is possible that the cardinality is not specified in `knora-admin`. This stems from a time, where I wasn't comfortable adding changes to then `knora-base`. Since it is optional, we can add it. Would you like to do it in an existing PR?
